src/main/java/games/strategy/triplea/image/BlendComposite.java

```diff
@@ -172,8 +172,9 @@ private static Blender getBlenderFor(final BlendComposite composite) {
                   Math.min(255, src[3] + dst[3])};
             }
           };
+        default:
+          throw new IllegalArgumentException("Blender not implemented for " + composite.getMode());
```

Comment:
Calling `name()` was unnecessary here as `toString()` will return the enum value name by default.